entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+# Remove a potentially pre-existing server.pid for Rails.
+rm -f /myapp/tmp/pids/server.pid
```

Comment:
Is this copypasta?

Reply:
Mannacotti - Thanks for the catch.

Reply:
Changed to /manifold.  Also addressed some S3 and Google API issues, and created a script to clean up and restore the database from data dump.